assets/queries/k8s/tiller_deployment_is_accessible_within_cluster/query.rego

```diff
@@ -0,0 +1,97 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    
+    isTiller(document)
+
+    result := isAccessible(document)
+}
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    
+    isTillerTemplate(document)
+
+    result := isAccessibleTemplate(document)
+}
+############################################################
+isTiller(document) {
+    document.spec.containers
+    checkMetadata(document.metadata)
+}
+
+isTillerTemplate(document) {
+    document.spec.template.spec.containers
+    checkMetadata(document.spec.template.metadata)
+}
+
+checkMetadata(metadata) {
+    contains(metadata.name,"tiller")
+}
+
+checkMetadata(metadata) {
+    object.get(metadata.labels,"app","undefined") == "helm"
+}
+
+checkMetadata(metadata) {
+    contains(object.get(metadata.labels,"name","undefined"),"tiller")
+}
+##############################################################
+isAccessible(document) = result {
+    container := document.spec.containers[j]
```

Comment:
We will add metadata information in order to be able to improve our searchKey

metadata := document.metadata

Reply:
address changes